docs/downloads/README.md

```diff
@@ -31,7 +31,8 @@ SkyWalking Nginx LUA agent requires SkyWalking APM 7.0+.
 | Date | Version| | Downloads |
 |:---:|:--:|:--:|:--:|
 | Dec. 10th, 2019 | 0.1.0 | Source code| [[src]](https://www.apache.org/dyn/closer.cgi/skywalking/cli/0.1.0/skywalking-cli-0.1.0-src.tgz) [[asc]](https://downloads.apache.org/skywalking/cli/0.1.0/skywalking-cli-0.1.0-src.tgz.asc) [[sha512]](https://downloads.apache.org/skywalking/cli/0.1.0/skywalking-cli-0.1.0-src.tgz.sha512)|
-| | | Binary Distribution (MacOS, Linux, Windows)| [[tgz]](https://www.apache.org/dyn/closer.cgi/skywalking/cli/0.1.0/skywalking-cli-0.1.0-bin.tgz) [[asc]](https://downloads.apache.org/skywalking/cli/0.1.0/skywalking-cli-0.1.0-bin.tgz.asc) [[sha512]](https://downloads.apache.org/skywalking/cli/0.1.0/skywalking-cli-0.1.0-bin.tgz.sha512)|
+| Mar. 20th, 2020 | 0.2.0 | Source code| [[src]](https://www.apache.org/dyn/closer.cgi/skywalking/cli/0.2.0/skywalking-cli-0.2.0-src.tgz) [[asc]](https://downloads.apache.org/skywalking/cli/0.2.0/skywalking-cli-0.2.0-src.tgz.asc) [[sha512]](https://downloads.apache.org/skywalking/cli/0.2.0/skywalking-cli-0.2.0-src.tgz.sha512)|
```

Comment:
I think we should remove 0.1.0 from dist and download link, as 0.2.0 is fully including all features of 0.1.0, right?

Reply:
0.1.0 is existing in the achieve repo automatically.